CHANGELOG.md

```diff
@@ -7,19 +7,13 @@ intended to be used with.
 Deprecated functionality will be retained for two releases after the release in
 which it is deprecated.
 
-## Unreleased
+## 1.7.0
 
 ### Added
 
 - A new method has been added to the `SwapSDK`, `requestDepositAddressV2`. This
   event simplifies deposit channel creation by accepting the quote that is
   returned by the SDK to set the correct assets, DCA parameters, and Boost fee.
-- A new method has been added to the `SwapSDK`,
```

Comment:
i don't think anyone will use this and we can remove it instead of continuing to maintain it

Reply:
smart